agent-install/edgeNodeFiles.sh

```diff
@@ -46,6 +48,7 @@ Required Environment Variables:
 
 Optional Environment Variables:
     PACKAGE_NAME: The base name of the horizon content tar file (can include a path). Default: $PACKAGE_NAME
+    HZN_EXCHANGE_USER_AUTH: The exchange credentials to use to publish artifacts in CSS. Must have access to publish to the IBM org. If not set, will use exchange root credentials.
```

Comment:
Will the IBM org always exist for people using OpenHorizon outside of IBM?

Reply:
Today, yes, the exchange automatically creates it. We are planning to change that, or at least call that org a different name (hopefully in the next release). At that point we'll have to change `edgeNodeFiles.sh`. But right now `edgeNodeFiles.sh` doesn't support pure open-horizon (it uses `cloudctl` and `oc` to access IAM). Hoping to have `edgeNodeFiles.sh` support pure open-horizon in the next release too.